src/lib/FormActions.js

```diff
@@ -0,0 +1,285 @@
+/**
```

Comment:
this can live within demosplan-core, as we will not build new ui using it.

Reply:
Done in:
https://github.com/demos-europe/demosplan-ui/pull/71/commits/d90277fdb90ee4349783e5c1e51137121a6e26c6
New PR in demosplan-core: https://github.com/demos-europe/demosplan-core/pull/666